metrics/default_metrics/luajit/init.lua

```diff
@@ -0,0 +1,75 @@
+local metrics = require('metrics')
+
+local has_mics_module, misc = pcall(require, 'misc')
+
+local gauges_storage = {}
+
+local LJ_PREFIX = 'lj_'
+
+local function prefix_name(name)
+    return LJ_PREFIX .. name
+end
+
+local function set_gauge(name, description, value, labels)
+    if (gauges_storage[name] == nil) then
+        gauges_storage[name] = metrics.gauge(prefix_name(name), description)
+    end
+
+    gauges_storage[name]:set(value, labels or {})
+end
+
+local function update()
+    local lj_metrics = misc.getmetrics()
```

Comment:
I suggest adding a ref to docs page describing luajit metrics

Reply:
I added a link to https://github.com/tarantool/doc/issues/1597